extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -595,7 +646,46 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
               visiblePercentageMin: 50,
               continuousTimeMin: 1,
             });
-    return superReturnValue;
+    return frameLoadPromise.then(result => {
+      if (this.isFluid_) {
+        if (!this.element.style.width) {
+          setStyles(this.element, {
+            width: computedStyle(this.win,
+                dev().assertElement(this.element.parentElement)).width,
```

Comment:
This still needs to be in measure/mutate phases. `#layoutCallback` is a time for mutates (it's already in a mutate phase), but this read causes the recalc.

Reply:
I don't follow. This is inside layoutCallback. Can you be a little more specific?

Reply:
Done.